### ISSUE TITLE
power(0, 0) exception from CAPD

### ISSUE THREAD

**Issue by soonhokong** [dreal/dreal3#242]
[benchmarks/drh/bouncing_ball/bouncing_ball.drh](https://github.com/dreal/dreal3/blob/master/benchmarks/drh/bouncing_ball/bouncing_ball.drh) generates exceptions. For example, here is one:
```
exception caught, "Interval error:  power(x, n): n == 0  and x contains 0.

   left=0  right=0
```

It turns out that the exception comes from CAPD and it seems that it's related to its derivation routine. When I replace `v^1` with `v`, the problem goes away. I'll do the following:

 - Do preprocessing to replace `x^n` with `x` when `n` is evaluated to `1`.
 - Report CAPD team the problem.

**Comment by soonhokong** [comment]
Also, need to update the benchmark to include the `v^2` case.

**Comment by soonhokong** [comment]
Send an email to CAPD team. Closed by 462ffb2.

**Comment by soonhokong** [comment]
CAPD team made a patch (adding simplification routines). I'll update `dreal-deps/capd4` repo soon.
